### ISSUE TITLE
website link on readme is not working yup.org

### ISSUE THREAD

**Issue by seanauriti** [notforhire/Yup#1]

**Comment by gwestwood** [comment]
yup.org is not the correct website, and since the developer has been missing for a while the real site (yupcrypto.com) is also down.  Please see https://github.com/YupCommunity/Yup where we are working to provide an updated community wallet and possibly website.
